iOS/chronreact/AppDelegate.m

```diff
@@ -19,6 +19,8 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
 {
   [BatchPush enableAutomaticDeeplinkHandling:NO];
   NSURL *jsCodeLocation;
+  
+  NSLog(@"%@", [BatchUser installationID]);
```

Comment:
This is just a print statement right? Mind getting rid of it if you don't need it anymore?